app/views/records/by_filter_task.html.erb

```diff
@@ -0,0 +1,26 @@
+<h1>Expected Result Patient List</h1>
+<div class="pull-right button-row">
+  <% if @measure && @measure.sub_id %>
+    <div class="btn-group">
+      <button type="button" class="btn btn-default dropdown-toggle" data-toggle="dropdown">Sub measures <span class="caret"></span>
+      </button>
+      <ul class="dropdown-menu">
+        <% submeasures = HealthDataStandards::CQM::Measure.where(hqmf_id: @measure.hqmf_id).sort_by(&:sub_id) %>
+        <% submeasures.each do |submeasure| %>
+          <li><%= link_to "#{submeasure.display_name} - #{submeasure.subtitle}", product_test_task_by_filter_task_path(@task.product_test, @task, sub_id: submeasure.sub_id),  method: :get %></li>
+        <% end %>
+      </ul>
+    </div>
+  <% end %>
+  <%= button_to good_results_task_path(@task), :method => :get, :class => "btn btn-default" do %>
+    <i class="fa fa-fw fa-download" aria-hidden="true"></i> Download HTML Patients
+  <% end %>
+</div>
+
+<% cache [@records, @measure, hide_patient_calculation?] do %>
+  <div class="row">
+    <div id="records_list" class="col-sm-12">
+      <%= render partial: 'records_list', locals: { records: @records } %>
```

Comment:
I am now getting `undefined local variable or method `measure' for #<#<Class:0x00007f806c7acf30>:0x00007f8064de4fe0>
Did you mean?  @measure` from this call when I navigate to by_filter_task.